packages/sitecore-jss-nextjs/src/services/graphql-sitemap-service.test.ts

```diff
@@ -503,6 +522,24 @@ describe('GraphQLSitemapService', () => {
     });
   });
 
+  it('should throw error if query returns nothing for a provided site name', async () => {
```

Comment:
The same test as on line 364? Was it expected to cover something different?

Reply:
My bad, second one should be testing export mode